tests/apps/courses/test_templates_organization_detail.py

```diff
@@ -24,6 +24,84 @@ class OrganizationCMSTestCase(CMSTestCase):
     End-to-end test suite to validate the content and Ux of the organization detail view
     """
 
+    def test_templates_organization_detail_open_graph_description_placeholder_description(
+        self,
+    ):
+        """
+        An opengraph description meta should be present if the organization description
+        placeholder is set.
+        """
+        organization = OrganizationFactory()
+        page = organization.extended_object
+
+        # Add a description to an organization
+        placeholder = organization.extended_object.placeholders.get(slot="description")
+        add_plugin(
+            language="en",
+            placeholder=placeholder,
+            plugin_type="CKEditorPlugin",
+            body="A longe description of the organization  ",
```

Comment:
This is where you should have html text no?

Reply:
Yes, sorry. I've missed that.